server/tests/phac.test.js

```diff
@@ -110,16 +109,6 @@ describe('Test phac-servicebc queries and endpoints', () => {
     expect(itemsToSend.length).toEqual(count - testTargets.length);
   });
 
-  it('Send unsuccessful Service BC transactions again', async () => {
```

Comment:
I asked in the other PR too, just missing context on why this was removed? :)

Reply:
Because it references the old cronjob structure. We need to create a ticket to test the lambda functions as it is now.

Reply:
Gotcha :)